themes/digital.gov/src/scss/new/_service-contact.scss

```diff
@@ -85,7 +85,7 @@
       }
     }
 
-    .authors-list-short {
+    .authors-list--inline {
       @include u-margin(0);
     }
   }
```

Comment:
Do we need this page override?

Reply:
Yes, with the new `authors-list--inline` variant class, we need this:

<img width="1228" alt="Screenshot 2024-07-02 at 2 11 21 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/6679ffdd-9d2d-41a9-b3e1-b49be37aec63">